test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -1100,3 +1100,51 @@ test_tsql_bubble_up_newline_after_fix:
   configs:
     core:
       dialect: tsql
+
+
+test_fail_templated_block_unindented_in_from_clause:
+  fail_str: |
+    SELECT
+        {% if true %}
+            1 AS col
+        {% else %}
+            2 AS col
+        {% endif %}
+    FROM
+    {% if true %}
+        a
+    {% else %}
+        b -- won't be fixed as it's in else block
+    {% endif %} -- won't indent, as it is outside from block
+    WHERE x = y
+
+  fix_str: |
+    SELECT
+        {% if true %}
+            1 AS col
+        {% else %}
+            2 AS col
+        {% endif %}
+    FROM
+        {% if true %}
+            a
+        {% else %}
+        b -- won't be fixed as it's in else block
+    {% endif %} -- won't indent, as it is outside from block
```

Comment:
Not sure I like this. While I agree it's difficult to make the `else` clause indent, surely the `{% endif %}` should be recognised as part of the `FROM` block?